.github/workflows/ci.yaml

```diff
@@ -107,13 +107,17 @@ jobs:
       - name: Build Docker image
         run: make -C newrelic-k8s-metrics-adapter image-push IMAGE_REPO=localhost:5000/newrelic-k8s-metrics-adapter
 
+      - name: Add newrelic repository
+        run: helm repo add newrelic https://helm-charts.newrelic.com
+
       - name: Create Helm release
         env:
           KUBECONFIG: ./newrelic-k8s-metrics-adapter/kubeconfig
           NEWRELIC_API_KEY: "${{ secrets.NEWRELIC_API_KEY }}"
           NEWRELIC_ACCOUNT_ID: "${{ secrets.NEWRELIC_ACCOUNT_ID }}"
         run: |
           make -C newrelic-k8s-metrics-adapter generate-local-values
+          helm dependency build ./helm-charts-newrelic/charts/newrelic-k8s-metrics-adapter
```

Comment:
Why are we updated dependencies for  `helm-charts-newrelic` chart?

We should have it locally and that chart has no dependencies This should not be needed, right?

Reply:
It's needed in order to get the common-library

Reply:
If not it was failing